src/data_info/clients/async_tasks.clj

```diff
@@ -68,49 +65,48 @@
 
 (defn run-async-thread
   [async-task-id thread-function prefix]
-  (otel/with-span [outer-span ["run-async-thread" {:kind :producer :attributes {"async-task-id" (str async-task-id)}}]]
-    (let [^Runnable task-thread (fn []
-                                  (with-open [_ (otel/span-scope outer-span)]
-                                    (otel/with-span [s ["async thread" {:kind :consumer :attributes {"async-task-id" (str async-task-id)}}]]
-                                      (thread-function async-task-id))))]
-      (.start (Thread. task-thread (str prefix "-" (string/replace async-task-id #".*/tasks/" "")))))
-    async-task-id))
```

Comment:
It looks like this function should still return `async-task-id`.
See `data-info.services.rename/move-paths` for example.

Reply:
Ah, good catch! I'll fix it. 👍